internal/cache/noop/noop_cache.go

```diff
@@ -0,0 +1,25 @@
+package noop
+
+import (
+	"context"
+
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+)
+
+type Cache struct{}
+
+func (_ *Cache) FindAccountId(_ context.Context, _, _ string) (*models.Account, bool) {
+	return nil, false
+}
+
+func (_ *Cache) SetAccountId(_ context.Context, _, _ string, _ *models.Account) {
+	// noop
+}
+
+func (_ *Cache) FindAppTypeId(_ context.Context) (string, bool) {
+	return "", false
+}
+
+func (_ *Cache) SetAppTypeId(_ context.Context, _ string) {
+	// noop
+}
```

Comment:
I am missing something.
 why do we have the same functions name here and internal/cache/memcache/mem_cache.go ? 
is it supposed to be an interface? I would imagine to see the functions implemented under Redis or memcache

Reply:
There is an interface, yes. See the `cache_interface.go` file, there are three implementations and a decorator that is only meant to deliver hit metrics (only for account):

https://en.wikipedia.org/wiki/Decorator_pattern